docs/apis-tools/frontend-development/introduction-to-task-applications.md

```diff
@@ -0,0 +1,141 @@
+---
+id: introduction-to-task-applications
+title: "Introduction to task applications"
+description: "Task applications are the interface between humans and Camunda processes to orchestrate human work."
+---
+
+import TasklistProcessesImg from './img/tasklist-processes.png';
+import TasklistDetailsImg from './img/tasklist-details.png';
+import TaskAssignmentImg from './img/task-assignment.png';
+import TaskLifecycleImg from './img/task-lifecycle.png';
+import styles from "./styles.module.css";
+
+Task applications are the interface between humans and Camunda processes to orchestrate human work. Learn key concepts of the architecture of task applications before you build your own.
+
+:::note
+If you are not familiar with the idea of human task management itself, read the [introduction to human task management](/components/best-practices/architecture/understanding-human-tasks-management.md) first.
+:::
+
+## What are task applications?
+
+Task applications are end-user applications that allow humans to perform work orchestrated with a process. A [user task](/components/modeler/bpmn/user-tasks/user-tasks.md/#user-task-forms) (also referred to as a **human task**) represents a single **work item** to be performed by an individual or a group. The jobs of a task application include:
+
+- Listing available tasks and allowing users to select a task to work on.
+- Providing filter and search options for users so they can more easily find the right next task to work on.
+- Presenting the selected task and an interface for completing the task, usually via a form.
+- Providing an interface to create new tasks, e.g. by starting a new process.
+- Provide insight into the progress of work tasks, including processes and cases.
```

Comment:
[Suggestion] "Offering insights into the progress of work tasks, encompassing processes and cases.""